lms/product/plugin/misc.py

```diff
@@ -0,0 +1,26 @@
+class MiscPlugin:
+    """
+    Plugin for product differences that don't have a better abstractions around them.
+
+    Works as a placeholder location to centralize LMS differences here without the need to create
+    bigger abstractions before it's clear what that abstraction should look like.
```

Comment:
:+1: I think this is a really important thing to note. Otherwise we'll end up backronyming this into the "Multi Instructure Service Configurator" or something one day.